### PR TITLE
(test) Add specs to vitals header test

### PR DESCRIPTION
**Pull request by nanfuka**
## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide).
- [x] My work includes tests or is validated by existing tests.

## Summary

This PR adds specs to the vitals header test suite that assert whether values get automatically flagged as abnormal or not.

### CODE REVIEW

**Comment by nanfuka** [comment]
@denniskigen kindly review

File: packages/esm-patient-vitals-app/src/vitals/vitals-header/vitals-header.test.tsx [hunk 1]
```diff
@@ -106,6 +106,69 @@ describe('VitalsHeader: ', () => {
...
+
+    const expandButton = screen.getByTitle(/ChevronDown/);
+    userEvent.click(expandButton);
+
+    expect(screen.queryByTitle(/abnormal value/i)).not.toBeInTheDocument();
+  });
+
+  it('renders a header with a pink backgroung when the diastolic value falls below the configured low normal range', async () => {
+    mockFhirVitalsResponse.entry[3].resource.valueQuantity.value = 69;
+
```
Comment (99sumedha) [inline packages/esm-patient-vitals-app/src/vitals/vitals-header/vitals-header.test.tsx:129]: Hi @nanfuka can you remove the extra line in 129

File: packages/esm-patient-vitals-app/src/vitals/vitals-header/vitals-header.test.tsx [hunk 2]
```diff
@@ -106,6 +106,69 @@ describe('VitalsHeader: ', () => {
...
+    await waitForLoadingToFinish();
+
+    const expandButton = screen.getByTitle(/ChevronDown/);
+    userEvent.click(expandButton);
+
+    expect(screen.queryByTitle(/abnormal value/i)).not.toBeInTheDocument();
+  });
+
+  it('renders a header with a pink backgroung when the diastolic value falls below the configured low normal range', async () => {
```
Comment (denniskigen) [inline packages/esm-patient-vitals-app/src/vitals/vitals-header/vitals-header.test.tsx:127]:
```suggestion
  it('flags abnormal values that lie outside of the provided reference ranges', async () => {
```

File: packages/esm-patient-vitals-app/src/vitals/vitals-header/vitals-header.test.tsx (hunk 1 above)
Comment (denniskigen) [reply packages/esm-patient-vitals-app/src/vitals/vitals-header/vitals-header.test.tsx:129]: A good pattern for writing unit tests is the [arrange-act-assert pattern](https://docs.telerik.com/devtools/justmock/basic-usage/arrange-act-assert). The use of a newline between the various sections of the test is a good practice for creating visual hierarchy and separation.

File: packages/esm-patient-vitals-app/src/vitals/vitals-header/vitals-header.test.tsx (hunk 2 above)
Comment (denniskigen) [reply packages/esm-patient-vitals-app/src/vitals/vitals-header/vitals-header.test.tsx:127]: This spec and the ones that follow it all seem to make the same assertion. It's probably better to combine them into one spec and make the different assertions within it.

**Review by denniskigen — COMMENTED** [review]
Good work overall, @nanfuka. I've added a few suggestions about organising your code.

File: packages/esm-patient-vitals-app/src/vitals/vitals-header/vitals-header.test.tsx (hunk 2 above)
Comment (denniskigen) [reply packages/esm-patient-vitals-app/src/vitals/vitals-header/vitals-header.test.tsx:127]: The other assertion could read something like:

```javascript
it(does not flag normal values that lie within the provided reference ranges, async () => {
```

**Review by denniskigen — APPROVED** [review]
LGTM! Thanks, @nanfuka.

**Pull request merged**